api/storage_service.cc

```diff
@@ -795,7 +795,7 @@ void set_storage_service(http_context& ctx, routes& r, sharded<service::storage_
     ss::get_keyspaces.set(r, [&ctx](const_req req) {
         auto type = req.get_query_param("type");
         if (type == "user") {
-            return ctx.db.local().get_non_system_keyspaces();
+            return ctx.db.local().get_non_internal_keyspaces();
```

Comment:
I suggest to simply calling it `get_user_keyspaces`.
The call to `is_internal_keyspace` is just an implementation detail :)